tests/Jackalope/PropertyTest.php

```diff
@@ -14,12 +12,12 @@ public function provideGetNode()
             array(
                 array('1234-1234', '4321-4321'),
                 array(),
-                'Could not find one or more referenced nodes: "1234-1234", "4321-4321"',
+                'Internal Error: Could not find one or more referenced nodes: "1234-1234", "4321-4321". If the referencing node is a frozen version, this can happen, otherwise it would be a bug.',
```

Comment:
Really? The tests were failing for me, because it seems like the exception message has changed.